include/sys/sdt.h

```diff
@@ -55,15 +51,9 @@
  * twice, so it should not have side effects (e.g. something like:
  * "return (SET_ERROR(log_error(EINVAL, info)));" would log the error twice).
  */
-#define	SET_ERROR(err) \
-	(trace_zfs_set__error(__FILE__, __func__, __LINE__, err), err)
-
-#else
-
 #undef SET_ERROR
-#define	SET_ERROR(err) (err)
-
-#endif /* HAVE_DECLARE_EVENT_CLASS */
+#define	SET_ERROR(err) \
+	(__dprintf(__FILE__, __func__, __LINE__, "error %lu", err), err)
```

Comment:
You're right, this is #6120 I happened to test build it on CentOS 7.  The Kernel.org Built-in builder would have also enabled these tracepoints and I see it did build fine.  Nevermind!